internal/npm_install/generate_build_file.ts

```diff
@@ -51,6 +51,8 @@ const PUBLIC_VISIBILITY = '//visibility:public';
 
 let LEGACY_NODE_MODULES_PACKAGE_NAME = '$node_modules$';
 
+const WORKSPACE_REROOTED_PATH = '_';
```

Comment:
strangely descriptive name for this constant that's one character, and unused?

Reply:
was used at one point, does that count?